core/templates/dev/head/pages/exploration_editor/loading_modal_directive.html

```diff
@@ -0,0 +1,27 @@
+<script type="text/ng-template" id="modals/loading">
```

Comment:
Let's call this modals/reloadingEditor. There's a lot of "loading" going on in different places so I think a different name is better.

Also, let's call the file editor_reloading_modal_directive.html.